networking/aws_load_balancer_operator/aws-load-balancer-operator-release-notes.adoc

```diff
@@ -13,6 +13,11 @@ These release notes track the development of the AWS Load Balancer Operator in {
 
 For an overview of the AWS Load Balancer Operator, see xref:../../networking/aws_load_balancer_operator/understanding-aws-load-balancer-operator.adoc#aws-load-balancer-operator[AWS Load Balancer Operator in {product-title}].
 
+[id="aws-load-balancer-operator-release-notes-1.1.z"]
+== AWS Load Balancer Operator 1.1.z
```

Comment:
Do we know the version number being released? If so, we need the actual version number here and in the advisory notice. I see that it states "1.1.z" in the erratum, but I sincerely doubt that that's the actual version number.

@alebedev87 What version is this?

Reply:
@ShudiLi can you answer?

Reply:
I think it should be 1.1.1.
[aws-load-balancer-operator-bundle-container-1.1.1-4](https://brewweb.engineering.redhat.com/brew/buildinfo?buildID=2864555)

Reply:
Yes, we shipped `1.0.1` and `1.1.1` versions for the rapid reset CVE fix on 30th of January.